.github/workflows/weekly-enviroment.yml

```diff
@@ -0,0 +1,126 @@
+name: Weekly environment deployment
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to deploy'
+        type: choice
+        options:
+          - weekly environment
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'INFO'
+        type: choice
+        options:
+          - TRACE
+          - DEBUG
+          - INFO
+          - WARN
+          - ERROR
+
+env:
+  WORKING_DIR: deploy/weekly-environment
+  SCRIPTS_DIR: deploy/weekly-environment/scripts/benchmarks/kspm_vanilla
+  TF_VAR_ec_api_key: ${{ secrets.WEEKLY_ENVIRONMENT_EC_API_KEY }}
```

Comment:
Who is the user behind this key?

Reply:
Currently its my user, we have two tickets related:

1. Short term solution - https://github.com/elastic/security-team/issues/6017, add the entire team to have permission on my account.

2. Long term solution - Create a team account - https://github.com/elastic/security-team/issues/5921